### ISSUE TITLE
What things are we looking at ?

### ISSUE THREAD

**Issue by seanxduan** [seanxduan/Data-III-Project-3#1]
EPA seems like our best threshold material to consider and see if our model is doing well or poorly.

We can consider effects of penalties on EPA and see if there's interaction by team?

We have enough data as well that we can create an outright holdout and training data sets for comparisions.

**Comment by reynolds-david** [comment]
**Feature engineering**
Game data: 
- Month
- Thursday night, Sunday noon, Sunday afternoon, Sunday night, or Monday night game

Player data: 
- Is the player from a Power Five program?
- BMI
- Age
- Average speed
- Average acceleration
- Average distance

Play data:
- Separate features from personnelO for number of RBs, TEs, and WRs
- Separate features from personnelD for number of DEs, LBs, and DBs
- Which side of the field the ball is on (own vs. other team's)
- Whether the defense is losing, winning, or tied
- Two-minute warning binary
- Home or away binary
- Score difference (defense - offense)
- Average player info

We could aggregate all of the play data by game and use completion percentage, number of sacks, etc. to predict something.

**Question 1: Can we predict whether or not a DPI will be called on the play (could also predict number of DPIs in a game)?**
Logistic regression, LDA, QDA, KNN, trees, SVMs, neural networks

**Question 2: Can we predict a player's performance on defense given their BMI, average speed, average acceleration, game information, etc.?**
Not sure how we could do this because there doesn't really seem to be a metric for measuring individual player performance, but this is essentially one of the example questions.

**Question 3: Can we predict the outcome of the pass play?**
Multinomial logistic regression, LDA, QDA, KNN, trees, SVMs, neural networks

I think we should try to find a way to use some unsupervised learning methods as well. Maybe cluster analysis with teams or players. Maybe there is a way we could use the player tracking data to do some spatial data analysis.

There's also no reason why we can't use outside data here.

**Comment by seanxduan** [comment]
Potential Questions: How does total mass affect the success of a given play defensively? Consider if the total weight on a team has much goal for success either way.

How does performance change when games are in 'garbage  time', a preset ratio of score differential and remaining clock time?

**Comment by seanxduan** [comment]
As an addendum to total mass, curious about how BMI data affects success. Avg BMI over both offense and defense, or perhaps just total BMI for a given set of key elements (offense or defense line only?). Interested in understanding if BMI truly does fall apart for elite athletic performance.

**Comment by seanxduan** [comment]
Perhaps see that if garbage time scoring is actually inflated or deflated, see if we can perhaps weight stats created during this time and see if we can weigh it more accurately and gain value from it.
